src/bundle/Resources/public/scss/_icons.scss

```diff
@@ -10,6 +10,10 @@
         width: 1.5rem;
         height: 1.5rem;
         fill: $ez-color-primary;
+
+        &--dark {
+            fill: $ez-secondary-ground-dark
```

Comment:
Could you please change it to `$ez-color-secondary`. Thanks!

Reply:
Of course, done

Reply:
I would also change the CSS class name to `ez-icon--secondary`